gempython/utils/db/settings.py

```diff
@@ -0,0 +1,17 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "gempython.utils.db.settings.local")
+SECRET_KEY = 'ef7$_)r0$#(x)ovd^m_6_bevsonkaxalvrhds9@ilb_p0o&-xn'
+
+INSTALLED_APPS = [
+    'gempython.utils.ldqm_db',
+    ]
+DATABASES = {
+    'default': {
+        'ENGINE':  'django.db.backends.mysql',
+        'NAME':     'ldqm_tif_qc8_db',
+        'PORT':     3306,
+        'HOST':     'cosmicstandtif',
+        'USER':     'gemdaq',
+        'PASSWORD': 'gemdaq',
+    }
+}
```

Comment:
I implemented the changes for `db` subdir. 
As for the DB_NAME - do you have the correct one on top of your head? I didn't change the one from the TIF because the main operation PC is the same and I assume it still has **that** DB =)